parser/sql/dialect/mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/MySQLStatementVisitor.java

```diff
@@ -1034,8 +1034,7 @@ public final ASTNode visitCastFunction(final CastFunctionContext ctx) {
                 result.getParameters().add((ColumnSegment) expr);
             } else if (expr instanceof LiteralExpressionSegment) {
                 result.getParameters().add((LiteralExpressionSegment) expr);
-            }
-            else if (expr instanceof FunctionSegment) {
+            } else if (expr instanceof FunctionSegment) {
```

Comment:
Hi @Afsalmc, does this change has sql parser test case cover?

Reply:
@strongduanmu No. I haven't made any changes to the test cases.

Reply:
What I mean is that after modifying the logic of visit, it needs to add the corresponding case to test it. Additionally, something went wrong with the CI, which also needed to be solved.